amcl/srv/SwapMap.srv

```diff
@@ -0,0 +1,5 @@
+ nav_msgs/OccupancyGrid map_msg
```

Comment:
I suggest this be called "map" rather than "map_msg" -- we already know it is a message.